externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/session/SparkSQLSessionManager.scala

```diff
@@ -78,7 +79,7 @@ class SparkSQLSessionManager private (name: String, spark: SparkSession)
       protocol,
       user,
       password,
-      ipAddress,
```

Comment:
Seems the ipAddress of the kyuubi server is also useful if we deploy in ha. We can get the information which Kyuubi server opened the engine session.

Reply:
> Seems the ipAddress of the kyuubi server is also useful if we deploy in ha. We can get the information which Kyuubi server opened the engine session.

Yes, but the current usage is more inclined to clientip, do we need to add an extra ip field?

Reply:
We can do this in a new PR, it's better to show client ip at UI side . And just record the kyuubi server ipAddress in event log.

Reply:
OK, I'll record an issue first.